src/pages/History/History.tsx

```diff
@@ -90,7 +90,7 @@ export const History = () => {
       <Tabs index={slideIndex} onChange={handleSlideIndexChange} variant='unstyled' pt={0} isLazy>
         <Box className='history-tabs-header' borderBottomWidth={1} borderColor='border.base'>
           <TabList px={4}>
-            <CustomTab>{translate('common.activity')}</CustomTab>
+            <CustomTab>{translate('navBar.activity')}</CustomTab>
```

Comment:
as sait previously, we probably don't want to change this as it will trigger some translations updates we could avoid, wdyt @firebomb1